load_average/load_average.tf

```diff
@@ -0,0 +1,71 @@
+resource "datadog_monitor" "load_average_1" {
+  count   = "${var.monitor_enabled}"
+  name    = "High load average 1 ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High load average 1 last ${var.load_average["time"]} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.public_ip}"
+  query   = "avg(last_${var.load_average["time"]}):avg:system.load.1{host:${data.aws_instance.monitored.instance_id}} by {host} > ${var.load_average["critical_threshold"]}"
+
+  thresholds {
+    ok       = "${var.load_average["ok_threshold"]}"
+    warning  = "${var.load_average["warning_threshold"]}"
+    critical = "${var.load_average["critical_threshold"]}"
+  }
+
+  renotify_interval = "${var.renotify_interval}"
+  new_host_delay    = "${var.new_host_delay}"
+  notify_no_data    = "${var.notify_no_data}"
+
+  silenced {
+    "*" = "${var.monitor_silenced}"
+  }
+
+  tags = "${var.datadog_monitor_tags}"
+}
+
+resource "datadog_monitor" "load_average_5" {
+  count   = "${var.monitor_enabled}"
+  name    = "High load average 5 ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High load average 5 last ${var.load_average["time"]}m on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.public_ip}"
+  query   = "avg(last_${var.load_average["time"]}):avg:system.load.5{host:${data.aws_instance.monitored.instance_id}} by {host} > ${var.load_average["critical_threshold"]}"
+
+  thresholds {
+    ok       = "${var.load_average["ok_threshold"]}"
+    warning  = "${var.load_average["warning_threshold"]}"
+    critical = "${var.load_average["critical_threshold"]}"
+  }
+
+  renotify_interval = "${var.renotify_interval}"
+  new_host_delay    = "${var.new_host_delay}"
+  notify_no_data    = "${var.notify_no_data}"
+
+  silenced {
+    "*" = "${var.monitor_silenced}"
+  }
+
+  tags = "${var.datadog_monitor_tags}"
+}
+
+resource "datadog_monitor" "load_average_15" {
+  count   = "${var.monitor_enabled}"
+  name    = "High load average 15 ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High load average last 15 last ${var.load_average["time"]} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.public_ip}"
+  query   = "avg(last_${var.load_average["time"]}):avg:system.load.15{host:${data.aws_instance.monitored.instance_id}} by {host} > ${var.load_average["critical_threshold"]}"
```

Comment:
I want to add a variable called `selector` that is required. The user must specify this. An example of the selector might be what you have here, but we cannot fix this to instances. Monitors should be able to work with tags. This will support that.